src/main/java/seedu/taskman/storage/XmlSerializableTaskMan.java

```diff
@@ -66,20 +71,37 @@ public UniqueActivityList getUniqueActivityList() {
                 //TODO: better error handling
             }
         }
+        for (XmlAdaptedEvent q : events) {
```

Comment:
Try not to use q if possible. Something more descriptive like event would be nice.